### ISSUE TITLE
Duplicate alt needs refinement

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#987]
Since the rules were ported over to JavaScript, there have been reports of the [Duplicate Alternative Text](https://equalizedigital.com/accessibility-checker/duplicate-alternative-text/) warning incorrectly flagging on SVG images.

A [user reported it in our Facebook Group](https://www.facebook.com/groups/wordpress.accessibility/posts/1472101123955850/?__cft__[0]=AZXriHbc8RrfYxfCSeLz4dj8kwySFFw2ePw7LBnxURo72R_KNda95yfgV2RfZYrfH9Nhjv1JbriyHQYY8zgOiNI0D42y6WvmjSkGQF1QdiadAcdUJvg8_homqZ0yyu_2Rl1wFbWAVCM7LbZx2XANjFKKdzu04gsYaI9z6k_PKeuQYqyAgiScN6heePPqLotoIfo&__tn__=%2CO%2CP-R) for the logos on these two websites:
- https://www.wheatlandsmetro.org/
- https://peacefulridgemd.com/

This is an example of the image being flagged:
`<img width="1" height="1" src="https://peacefulridgemd.com/wp-content/uploads/2024/04/PRMD-logo.svg" class="custom-logo svg-logo-image" alt="Peaceful Ridge Metropolitan District logo" decoding="async">`

This is also being flagged now on XeoMatrix for the linked logo.

Code saved in EDAC:
`<img width="261" height="70" src="https://www.xeomatrix.com/wp-content/uploads/2023/06/Xeomatrix_Name_White-1.svg" class="attachment-full size-full wp-image-5763" alt="XeoMatrix">`

Code loaded:
```
<a href="https://www.xeomatrix.com">
   <img width="261" height="70" data-src="https://www.xeomatrix.com/wp-content/uploads/2023/06/Xeomatrix_Name_White-1.svg" class="attachment-full size-full wp-image-5763 ls-is-cached lazyloaded" alt="XeoMatrix" src="https://www.xeomatrix.com/wp-content/uploads/2023/06/Xeomatrix_Name_White-1.svg" style="--smush-placeholder-width: 261px; --smush-placeholder-aspect-ratio: 261/70;">
</a>
```

We discussed that this might be a result of this part of the rule logic:
> identical alt text on two different images on the same page

This needs some refinement to reduce manual checking burden, if possible. Here are some examples when duplicate alt should not flag:
- The same image (same image file URL) used multiple times on a page would be expected to have the same alt.
- Linked images going to the same URL would be expected to have the same alt (even if the file URL for the image is different, because the alt should not describe the image, but should, instead, describe where the link goes.

I also wonder if this part of the check should be split into its own rule apart from the checks that just look if the alt is redundant with captions or surrounding text, since how you fix it and other information would be different. I.e., when there are two images with the same alt, it would be ideal if the checker flagged them both in the code snippet or on the page.
